packages/syft/src/syft/client/domain_client.py

```diff
@@ -134,6 +136,11 @@ def upload_dataset(self, dataset: CreateDataset) -> SyftSuccess | SyftError:
                         syft_node_location=self.id,
                         syft_client_verify_key=self.verify_key,
                     )
+                    serialized: bytes = serialize(twin, to_bytes=True)
+                    size_mb: float = sys.getsizeof(serialized) / 1024 / 1024
+                    if size_mb < 16:
```

Comment:
Can we make this configurable so that the max size can be set in an environment variable for when we change database implementations and allow this to be configurable?

Reply:
Yes sure, thanks Madhava. It should be that way